misk-aws/src/main/kotlin/misk/jobqueue/sqs/SqsJobConsumer.kt

```diff
@@ -107,6 +101,27 @@ internal class SqsJobConsumer @Inject internal constructor(
       }.join()
     }
 
+    /**
+     * List containing arbitrary numbers. The size of the list is the number of receivers to use.
+     */
+    private fun receivers(): List<Int> {
+      val numReceiversPerPodForQueue = receiversPerPodForQueue()
+      val shouldUsePerQueueConfig = numReceiversPerPodForQueue < 0
+      if (shouldUsePerQueueConfig) {
```

Comment:
nit: For readability I would invert this if condition to reduce the overall amount of indentation
```
if (shouldUseNewConfig()) {
    return (1..numReceiversPerPodForQueue).toList()
}
return (1..numReceiversForQueue())
    .filter {
      // ...
     }
```